docs/_static/theme_overrides.css

```diff
@@ -15,3 +15,35 @@
   float: right;
   margin-left: 0.25rem;
 }
+
+:root {
+    /* dwave colors for convenience */
+    --dwave-color-ocean-blue: 42, 125, 225;
+    --dwave-color-dark-blue: 7, 76, 145;
+    --dwave-color-bright-blue: 3, 184, 255;
+
+    --dwave-color-leap-teal: 23, 190, 187;
+    --dwave-color-dark-teal: 0, 140, 130;
+    --dwave-color-bright-teal: 6, 236, 220;
+
+    --dwave-color-advantage-orange: 243, 120, 32;
+    --dwave-color-dark-orange: 175, 73, 4;
+    --dwave-color-bright-orange: 255, 161, 67;
+
+    /* theme overrides */
+    --pst-color-primary: 34, 34, 34;
+    --pst-color-link: var(--dwave-color-ocean-blue);
+    --pst-color-link-hover: var(--dwave-color-dark-blue);
+    /*--pst-color-inline-code: var(--dwave-color-advantage-sienna);*/
+    --pst-color-success: var(--dwave-color-leap-teal);
+    --pst-color-info: var(--dwave-color-ocean-blue);
+    --pst-color-warning: var(--dwave-color-advantage-orange);
+    /*--pst-color-danger: 220, 53, 69;*/
+    --pst-color-active-navigation: var(--dwave-color-ocean-blue);
+    --pst-color-navbar-link-hover: var(--dwave-color-ocean-blue);
+    --pst-color-navbar-link-active: var(--dwave-color-ocean-blue);
+    --pst-color-sidebar-link-hover: var(--dwave-color-ocean-blue);
+    --pst-color-sidebar-link-active: var(--dwave-color-ocean-blue);
+    --pst-color-toc-link-hover: var(--dwave-color-ocean-blue);
+    --pst-color-toc-link-active: var(--dwave-color-ocean-blue);
+}
```

Comment:
```suggestion
}
```
```suggestion
}



#navbar-main {
    /* we need to use !important because pydata-sphinx-theme does */
    /* though to be fair, that's because of one of their dependencies */
    /* Match sphinx-panels _colors.scss gray-100 */ 
    background-color: #f8f9fa !important;
}
```